python/tvm/relay/frontend/pytorch.py

```diff
@@ -458,7 +458,10 @@ def _impl(inputs, input_types):
         data = inputs[0]
 
         pool_size = _infer_shape(inputs[1])
-        strides = _infer_shape(inputs[2])
+        if inputs[2]:
+            strides = _infer_shape(inputs[2])
+        else:
+            strides = pool_size
```

Comment:
is this correct? we should use the default strides

Reply:
https://github.com/pytorch/pytorch/blob/17a5c677963dc3ecb7ff505585ed15eadaaf74ef/torch/nn/functional.py#L269-L270
According to the description of strides above, for avg_pool2d. Stride's  default value kernel_size, which should have the same size as pool_size here

Reply:
ok